lua/flash/plugins/remote.lua

```diff
@@ -45,9 +45,15 @@ function M.jump(opts)
   opts = Config.get(opts, { mode = "remote" }, {
```

Comment:
Also, I think `opts` and `{mode = "remote"}` should be swapped here since it seems like the `remote` mode is overriding the provided `opts`

Reply:
yes indeed. will change that